Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM ruby:2.7.0-alpine
+FROM ruby:alpine3.15
```

Comment:
Maybe we should move to startback? I'm not sure. Alpine is possibly better. Any idea?

Reply:
Is that ruby 3.1 already? That's possibly worth a try.

Reply:
maybe @llambeau has a better opinion?